presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergPageSink.java

```diff
@@ -105,10 +106,10 @@ public class IcebergPageSink
     private long writtenBytes;
     private long systemMemoryUsage;
     private long validationCpuNanos;
+    private Table table;
 
     public IcebergPageSink(
-            Schema outputSchema,
-            PartitionSpec partitionSpec,
+            Table table,
```

Comment:
Can you help explain why we need these changes to `IcebergPageSink`, and why do we need to pass in the Iceberg `Table` when it seems we're just using the Presto `PrestoIcebergTable` implementation?

Reply:
The problem encountered when creating `IcebergFileWriter` in `IcebergPageSink` is that, among all the static methods provided by Iceberg Lib for building a `MetricsConfig` instance, we can only pass an iceberg `Table` object in order to specify the table's `schema`. Refer to the static methods in `MetricsConfig`.

However, at present, there isn't a way on the presto worker side for us to load an iceberg `Table` object through metadata manager. So we just built an iceberg `Table` interface's implementation that is only used for simple encapsulation of `schema`, `properties`, 'spec', and 'sortOrder'. Then pass to and use it in `IcebergPageSink`.